.circleci/config.yml

```diff
@@ -231,16 +231,17 @@ jobs:
       BROWSER: << parameters.browser >>
       BVER: << parameters.bver >>
     trigger-qe-tests:
+      executor: generic-executor
       docker:
         - image: circleci/node:latest
       steps:
         - run:
             name: Trigger QE tests
             command: |
-              curl --silent --output /dev/null -u USER_TOKEN: -X POST \
+              curl --silent --output /dev/null -u CIRCLECI_PERSONAL_API_TOKEN: -X POST \
               --header "Content-Type: application/json" \
-              -d '{"branch":"v${CIRCLE_TAG:0:1}","parameters":{"sdk_version": CIRCLE_TAG,"is_rc":true}}' \
-              $SDKS_QE_CIRCLECI_VOICE_JS_TESTS_ENDPOINT
+              -d '{"branch":"'v${CIRCLE_TAG:0:1}'","parameters":{"sdk_version":"'$CIRCLE_TAG'","is_rc":true}}' \
+              $SDK_JS_VIDEO_SLACK_WEBHOOK
```

Comment:
Is this variable in SDKSQE Context? Could you please move it if not the case?

Reply:
Yes, it's in context